tfjs-backend-webgpu/src/kernels/BatchMatMul_impl.ts

```diff
@@ -88,14 +88,15 @@ export function batchMatMulImpl({
   const intermediates: TensorInfo[] = [a3d, b3d];
 
   const batchDim = Math.max(batchDimA, batchDimB);
-
+  const batchAEqualOne = batchDimA === 1;
+  const batchBEqualOne = batchDimB === 1;
   const useVec4 = innerShapeA % 4 === 0 && outerShapeB % 4 === 0 &&
-      !transposeA && !transposeB && outerShapeB >= 32;
+      !transposeA && !transposeB;
   let program: WebGPUProgram;
   if (outerShapeA * outerShapeB <= 32) {
     program = new MatMulReduceProgram(
-        [batchDim, outerShapeA, outerShapeB], transposeA, transposeB, bias,
-        activation, preluActivationWeights);
+        [batchDim, outerShapeA, outerShapeB], batchAEqualOne, batchBEqualOne,
```

Comment:
Is it better to name these variables as `batchEqualOneA`, `batchEqualOneB`, which are similar as `transposeA` and `transposeB`

Reply:
I think `batchAEqualOne` is also easy to understand. If you don't have strong preference, can we just keep it unchanged?